### ISSUE TITLE
Wrong parsing for `if` block with +=

### ISSUE THREAD

**Issue by ntrel** [vlang/v#929]
**V version:** 
0.1.8
**OS:** 
https://vlang.ide.judge0.com/

**What did you do?**
```
fn hex(c byte) i64 {
	mut r := i64(0)
	if c <= `F` {r += 2}
}
```
**What did you expect to see?**
The code compile.
 
**What did you see instead?**
```
check()
next token = `EOF`
pass=2 fn=`hex`
panic: main.v:4
expected `}` but got `EOF`
```

**Comment by cytown** [comment]
You need this kind of format:
if xxx {
...
}

`if xxx {...}` is not supported in many languages.

**Comment by yep84** [comment]
You should try it:
```
fn hex(c byte) i64 {
	mut r := i64(0)
	if c <= `F` {
        r += 2
    }
    return r
}
```

**Comment by ntrel** [comment]
> if xxx {...} is not supported

`if c <= `F` {return r}` works, so this should too. If it's not supposed to work, then it's a bug that return works.

**Comment by medvednikov** [comment]
It's a bug with the parser. If `{` is on the same line, it doesn't work. Will fix.

**Comment by ntrel** [comment]
Same as #606 (this has the better title though ;-))

**Comment by medvednikov** [comment]
I can't reproduce this by the way.

```
fn hex(c byte) i64 {
	mut r := i64(0)
	if c <= `F` {r += 2}
}
```
compiles fine

**Comment by ntrel** [comment]
> compiles fine

Looks like this got fixed in 0.1.11, thanks!
